### PR TITLE
Feature/content search adgerdir page importer

### PR DESCRIPTION
**Pull request by stefanhar**
# \<Description\>

https://app.asana.com/0/1199123945323117/1199359619863901

## What

- Made search page use new link resolver
- Made link resolver accept empty contentType as frontpage
- Reworked sidebar tag code
- Made sidebar render adgerdirPage
- Made sidebar handle dynamic other content tags
- Made search page render adgerdir search results card
- Fixed sidebar counting wrong issue
- Removed news specific display code
- Added type as query parameter
- Removed typename param from page types
- Added adgerdir page to search engine import
- Changed search engine type resolution in api
- Added type aggregation to api

## Why

- To display adgerdirPage in other content in search results sidebar
- Cleanup old code

## Screenshots / Gifs

Screenshot 2020-11-26 at 17 10 44
Screenshot 2020-11-26 at 17 10 50
Screenshot 2020-11-26 at 17 10 56

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/web/screens/Search/Search.tsx [hunk 1]
```diff
@@ -142,15 +180,12 @@ const Search: Screen<CategoryProps> = ({
...
     title: item.title,
-    description: item.intro ?? item.seoDescription,
-    href:
-      item.__typename === 'AboutPage'
-        ? item.slug
-        : makePath(item.__typename, '[slug]'),
-    as: makePath(item.__typename, item.slug),
+    description: item.intro ?? item.seoDescription ?? item.description,
+    href: pathNames(activeLocale, item.__typename, [item.slug])?.href,
+    as: pathNames(activeLocale, item.__typename, [item.slug])?.as,
```
Comment (siggakogk) [inline apps/web/screens/Search/Search.tsx:188]: 👍 nice!

File: apps/web/screens/Search/Search.tsx [hunk 2]
```diff
@@ -255,93 +255,88 @@ const Search: Screen<CategoryProps> = ({
...
+                            )
+                          },
+                        )}
                       </Stack>
                     </SidebarAccordion>
                   </>
                 )}
               </Box>
             </Sidebar>
-            {totalNews > 0 && (
```
Comment (siggakogk) [inline apps/web/screens/Search/Search.tsx:294]: Do we want to show the other categories sidebar if it has no results?

**Review by siggakogk — APPROVED** [review]
minor comment otherwise LGTM

File: apps/web/screens/Search/Search.tsx (hunk 2 above)
Comment (stefanhar) [reply apps/web/screens/Search/Search.tsx:294]: This is valid, will fix in another PR since this got auto merged

**Pull request merged**